payments-core/src/main/java/com/stripe/android/googlepaysheet/GooglePayLauncherConfigureViewModel.kt

```diff
@@ -2,7 +2,7 @@ package com.stripe.android.googlepaysheet
 
 import androidx.lifecycle.ViewModel
 
-internal class GooglePaySheetConfigureViewModel : ViewModel() {
+internal class GooglePayLauncherConfigureViewModel : ViewModel() {
```

Comment:
ooc, does this need to be a viewModel, is it really just for the configure?

Reply:
I'm using it to keep track of args across Activity config changes.